radio/src/main_avr.cpp

```diff
@@ -129,6 +132,9 @@ void perMain()
   const char *warn = s_warning;
   uint8_t menu = s_menu_count;
 
+#if defined(LCD_ST7920)
+  if(0==lcdstate){//No need to redraw until lcdRefresh_ST7920(0) below completely refreshes the display.
```

Comment:
Thank you for reviewing. Would you be please more specific. Where should I define the macro?
I would also appreciate some guidance regarding the procedure from this point. Should I create another commit and pull request?